Formula/lib/libcoap.rb

```diff
@@ -24,26 +24,19 @@ class Libcoap < Formula
 
   def install
     system "./autogen.sh"
-    system "./configure", "--disable-examples", "--disable-manpages", *std_configure_args
+    system "./configure", "--disable-manpages", "--disable-doxygen", *std_configure_args
```

Comment:
We should either:
- install the `doxygen` documentation; or,
- remove the `doxygen` build dependency

See https://github.com/Homebrew/homebrew-core/blob/de440c1d877d7bfe90575b5ed0a9478b83fe08bb/Formula/lib/libcoap.rb#L21

Reply:
Good catch! I would tend towards removing the build dependency. Is there a particular standard to follow here?

Reply:
Yea, just remove it. If anyone misses it, they can add it back in a PR. Though it doesn't seem like anyone will, since we never installed it in the first place.